components/gitpod-db/src/typeorm/user-db-impl.ts

```diff
@@ -662,16 +663,13 @@ export class TypeORMUserDBImpl extends TransactionalDBImpl<UserDB> implements Us
 
     async findUserIdsNotYetMigratedToFgaVersion(fgaRelationshipsVersion: number, limit: number): Promise<string[]> {
         const userRepo = await this.getUserRepo();
-        const ids = (await userRepo
+        const users = await userRepo
             .createQueryBuilder("user")
-            .select(["id"])
-            .where({
-                fgaRelationshipsVersion: Not(Equal(fgaRelationshipsVersion)),
```

Comment:
@svenefftinge Can you share some insight into what's not working here, e.g. how the resulting query looks like?

Reply:
no, I wrote a test and verified it failed and then used the pattern we use in other queries as well to make the test pass.